### ISSUE TITLE
issues on RetroPie x86

### ISSUE THREAD

**Issue by meleu** [daeks/RetroPie-WebGui#33]
I tried to install yarman using your scriptmodule in my RetroPie x86 and the the packages php5 and php5-sqlite don't exist in the official Ubuntu/Linux Mint repositories...

Then I changed the script to install `php` and `php-sqlite` and it seems that yarman is able to run...

**Comment by meleu** [comment]
Ooops! Problems with dashboard in RetroPie x86:
![captura de tela de 2017-03-04 22-29-00](https://cloud.githubusercontent.com/assets/8508804/23583695/72fd1534-012a-11e7-8414-aa78aa022e6a.png)

**Comment by daeks** [comment]
Hmm, regarding the first point, I have only tested it on a pi. Have to check if ```php``` is also working on a pi, then we can change it :) or it needs some distinction between a pi and x86. 

Second point should be the same issue. Only tested on pi. I will add some exception handling to it.

**Comment by meleu** [comment]
Let me know if you need some log file to help the dashboard fixing.

Cheers.

**Comment by daeks** [comment]
if you want you could test with the sqlite branch and add a screenshot of the dashboard. I have changed a couple of panels on it.

**Comment by meleu** [comment]
Using the sqlite branch I always get this popup:
![captura de tela de 2017-03-06 23-24-28](https://cloud.githubusercontent.com/assets/8508804/23639089/312be1b2-02c4-11e7-8082-194bedfed087.png)

I tried to Close, Sync and Sync with that checkbox checked... No matter what I choose the popup closes and instantaneously comes back.

**Comment by daeks** [comment]
Interesting that the background is empty. That shouldn't be. 
Regarding the "sync behavior" I believe your path differs from the default "/home/pi/RetroPie/roms"? Therefore I have changed the setup process a bit :)

**Comment by daeks** [comment]
Regaring the empty dashboard, it looks like the problem is within the integrated php webserver... it is fixed now.

**Comment by meleu** [comment]
> Regarding the "sync behavior" I believe your path differs from the default "/home/pi/RetroPie/roms"? 

yeah, If you wanna make this tool available to other platforms than raspi you shouldn't hardcode the `pi` user homedir. But if you wanna limit it to raspi, the scriptmodule should specify the platforms to **NOT** install in `rp_module_flags` ([this forum topic](https://retropie.org.uk/forum/topic/4621/some-questions-about-retropie-script-modules) can be useful).

I make my tests/development in a x86 environment because it's on my laptop and I'm always carrying it with me, but I think it's useful for other users too.

I think the scriptmodule in the RetroPie-Setup PR is at a very embryonic stage. I'll try to make something similar to what I made for RetroPie-Manager and submit to the same branch of your Pull Request there.

**Comment by daeks** [comment]
Currently nothing is hardcoded :) but there is no config option for that yet. Defaults are set for raspi [here](https://github.com/daeks/yarman/blob/master/core/db/config.jdb#L15). I already changed that at the setup process. I will add an admin panel later on as well for configuring.

![image](https://cloud.githubusercontent.com/assets/3989440/23654537/62a80492-0331-11e7-8093-80a1affaf175.png)

I appreciate your support with the scriptmodule :) if it is required I can also close my PR there. As soon as it is merged in the official RetroPie-Setup I will remove the scriptmodule's in my repository as well.

**Comment by meleu** [comment]
> if it is required I can also close my PR there.

It's not exactly required, but it can be a good idea since it would be cleaner to @joolswills review the code later.
